docs/balance-sheet.md

```
@@ -173,9 +173,86 @@ Go 侧处理：正则 `/jQuery.*?\((.*)\);?$/s` 提取中间 JSON，再 `json.Un
+| 交易所       | 代码前缀 | orgId 示例               |
+| ------------ | -------- | ------------------------ |
+| 深交所主板   | `00`     | `000001` → `gssz0000001` |
+| 深交所创业板 | `30`     | `300750` → `gssz0300750` |
+| 上交所主板   | `60`     | `600036` → `gssh0600036` |
+| 上交所科创板 | `68`     | `688001` → `gssh0688001` |
+
+#### PDF 文件命名规则
+
+PDF 保存在 CNINFO 的静态 CDN 上，URL 格式：
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)